cypress/component/ganttchart.cy.js

```diff
@@ -71,7 +71,46 @@ describe('GanttChart', () => {
       .should('exist')
       .should('be.visible')
   })
-  it('filters by total time', () => {
+  it('filters by total time (firefox)', { browser: 'firefox' }, () => {
+    // see: https://on.cypress.io/mounting-vue
+    cy.mount(GanttChart, merge(mountOpts, {
+      props: {
+        jobs: [job1, job2],
+        timingOption: 'total'
+      },
+    }))
+    cy.get('.apexcharts-rangebar-area')
+      .first()
+      .invoke('attr', 'barWidth')
+      .should('eq', '161.1500015258789')
```

Comment:
Rather than test the bar size (which is browser-specific) in a component test, you can test the component computed property "`series`" directly in a unit test

Reply:
Agreed that browser specific tests are "icky".
Can you give me a hand creating the test? I'm struggling a bit with them